pram

```diff
@@ -286,7 +296,16 @@ main() {
 	fi
 
 	if [[ -n ${pr} ]]; then
-		wget -O "${tempdir}/all.patch" "${pr}" || die "Fetching patch failed"
+		if [[ -n ${api} && ${pr} == *://github.com/* ]]; then
+			pr=${pr/github.com/api.github.com\/repos}
```

Comment:
Will this work correctly when using an URL like `https://github.com/org/repo/commit/<hash>.patch`? I don't think it will, so you probably need to restrict the match further.

Reply:
Forgot that was an option. It looks like the API can manage it but it needs similar treatment with s/commit/commits/

Reply:
Would you want comments with links to the api? Or just a comments explaining this.

Reply:
Just discovered that `https://github.com/org/repo//pull/<pull id>/commits/<hash>.patch` is also a valid option...

Reply:
Yeah, we basically support anything that can accept a `.patch` suffix, and actually use them all. API links will probably be better.